README.md

```diff
@@ -70,21 +70,23 @@ are as follows:
 5. Create a file in the `model` directory named `main.smithy` with the following contents:
 
     ```
+    $version: "2"
+   
     namespace smithy.example
 
     service ExampleService {
-        version: "2022-01-01",
+        version: "2022-01-01"
         operations: [Echo]
     }
 
     operation Echo {
-        input: EchoInput,
-        output: EchoOutput,
+        input: EchoInput
```

Comment:
let's also update to use inline Input/output

Reply:
Updated. We could also extract `message: String` to a mixin, but I don't have a strong opinion on whether or not to do that